src/core/annotation.js

```diff
@@ -2072,6 +2078,63 @@ class ButtonWidgetAnnotation extends WidgetAnnotation {
     return newRefs;
   }
 
+  _getDefaultCheckedAppearance(params, type) {
+    const width = this.data.rect[2] - this.data.rect[0];
+    const height = this.data.rect[3] - this.data.rect[1];
+    const bbox = [0, 0, width, height];
+
+    // Ratio used to have a mark slightly smaller than the bbox.
+    const FONT_RATIO = 0.8;
+    const fontSize = Math.min(width, height) * FONT_RATIO;
+
+    // Char Metrics
+    // Widths came from widths for ZapfDingbats.
+    // Heights are guessed with Fontforge and FoxitDingbats.pfb.
+    let metrics;
+    let char;
+    if (type === "check") {
+      // Char 33 (2713 in unicode)
+      metrics = {
+        width: 0.755 * fontSize,
+        height: 0.705 * fontSize,
+      };
+      char = "\x33";
+    } else {
```

Comment:
Can we please use `} else if (type === "disc") {` here instead, and then add an additional branch below to ensure that the caller explicitly chooses a valid type?
```javascript
} else {
  unreachable(`_getDefaultCheckedAppearance - unsupported type: ${type}`);
}
```